src/cudadev/plugin-PixelTriplets/gpuFishbone.h

```diff
@@ -16,76 +19,107 @@
 
 namespace gpuPixelDoublets {
 
-  //  __device__
-  //  __forceinline__
-  __global__ void fishbone(GPUCACell::Hits const* __restrict__ hhp,
+  template <int hitsPerBlock = 1, int threadsPerHit = 1>
```

Comment:
Given that in this PR the template arguments are given explicitly in all callers, how necessary the default values are? I'm mostly concerned on the self-documentation of the code.

Reply:
I added them so the CPU version could be called without specifying `<1, 1>`, but forgot to remove them from the call.

I agree that we should remove either the default or the explicit arguments in the CPU case.